source/geometry_model/spherical_shell.cc

```diff
@@ -396,7 +396,6 @@ namespace aspect
                              "this geometry are 90, 180, and 360 in 2d; "
                              "and 90 and 360 in 3d. "
                              "Units: degrees.");
-
```

Comment:
why are you removing this line?

Reply:
hmm, interesting. I never realized we (also me if I look at other models) did that :s